### PR TITLE
lambda@generateURL

### PR DESCRIPTION
**Pull request by ayush0x00**

### CODE REVIEW

**Comment by ayush0x00** [comment]
Fixes a part of #34

File: lambdas/url_access_patterns/lambda_for_generateURL/index.js
Comment (danishjamal104) [inline lambdas/url_access_patterns/lambda_for_generateURL/index.js]: No need to stringify in the case of `API gateway`

Comment (ayush0x00) [reply lambdas/url_access_patterns/lambda_for_generateURL/index.js]: forgot to change it. Anything else. @lakshya-20 ?

File: lambdas/url_access_patterns/lambda_for_generateURL/index.js
Comment (lakshya-20) [inline lambdas/url_access_patterns/lambda_for_generateURL/index.js]: `PK` should be `FILE#${reqBody.f_timestamp}` and `SK` should be `URL#${timestamp}` according to modelling.
Have you done this intentionally?

**Review by lakshya-20 — CHANGES_REQUESTED** [review]

File: lambdas/url_access_patterns/lambda_for_generateURL/index.js
Comment (ayush0x00) [reply lambdas/url_access_patterns/lambda_for_generateURL/index.js]: my bad. will fix it.

File: lambdas/url_access_patterns/lambda_for_generateURL/index.js
Comment (danishjamal104) [inline lambdas/url_access_patterns/lambda_for_generateURL/index.js:46]: change the status code, in case of error

**Review by danishjamal104 — CHANGES_REQUESTED** [review]

File: lambdas/url_access_patterns/lambda_for_generateURL/index.js
Comment (lakshya-20) [reply lambdas/url_access_patterns/lambda_for_generateURL/index.js:46]: We have to give a static status code and it is to be done for all lambdas so we thought of opening a new issue for lambdas. So he has left it as it is.

Comment (danishjamal104) [reply lambdas/url_access_patterns/lambda_for_generateURL/index.js:46]: then why @lakshya-20  haven't you approved the PR?

Comment (danishjamal104) [reply lambdas/url_access_patterns/lambda_for_generateURL/index.js:46]: This must be addressed with reference to the issue #39

**Comment by ayush0x00** [comment]
Is there any changes to be made. If not please merge.

**Comment by danishjamal104** [comment]
> Is there any changes to be made. If not please merge.

I was waiting for @lakshya-20 's approval.

**Review by danishjamal104 — APPROVED** [review]

**Pull request merged**
